Retailcoder.VBE/UI/RubberduckUI.resx

```diff
@@ -381,6 +381,15 @@ Warning: All customized settings will be lost.  Your old file will be saved in '
   <data name="Rubberduck_AboutBuild" xml:space="preserve">
     <value>Version {0}</value>
   </data>
+  <data name="TodoMarkerBug" xml:space="preserve">
+    <value>BUG</value>
+  </data>
+  <data name="TodoMarkerNote" xml:space="preserve">
+    <value>NOTE</value>
+  </data>
+  <data name="TodoMarkerTodo" xml:space="preserve">
+    <value>TODO</value>
+  </data>
```

Comment:
Warning: ghost resx reappeared

Reply:
Not sure how to do this...However, https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-git-repo looks promising.   I lack the experience and confidence with GIT to 'just do it'...Thoughts/comments/recommendations?

I definitely can see the ghost file if I generate a list of files in my local repository.  But, the file is not present/visible in File Explorer.

Reply:
try using `git rm --cached` to prevent windows misinterpreting the given path and subsequently deleting the actual resx file.